cabal.project.freeze

```diff
@@ -5,23 +5,16 @@ constraints: any.Cabal ==3.6.3.0,
              any.OneTuple ==0.3.1,
              any.Only ==0.1,
              any.QuickCheck ==2.14.3,
-             QuickCheck -old-random +templatehaskell,
              any.StateVar ==1.2.2,
              any.adjunctions ==4.4.2,
              any.aeson ==2.0.3.0,
-             aeson -cffi +ordered-keymap,
              any.aeson-pretty ==0.8.9,
-             aeson-pretty -lib-only,
              any.ansi-terminal ==0.11.4,
-             ansi-terminal -example +win32-2-13-1,
```

Comment:
The script `scripts/freeze-cabal-to-stack-resolver.sh` generates these flags so maybe we should not remove them?
Or is the `nix`-ified version not generating them?

Reply:
My local version of cabal removed those and i dont really have a way to run the exact version as on CI. Since we don't use the freeze file for anything anyway (as far as i know) and solely depend on stack.yaml in CI, it doesn't matter much IMO and in the next haskell backend ci update job it will likely put those back anyway...

Reply:
OK fair enough, if we run into problems with the script we'll notice it soon enough.
I thought we were using the freeze file in the nix build (which uses `cabal`)...

Reply:
We were before, but I have switched us back to using stack.yaml/stackage as source of truth for nix.